### ISSUE TITLE
DeepEnhancer: Predicting enhancers by convolutional neural networks

### ISSUE THREAD

**Issue by alxndrkalinin** [greenelab/deep-review#343]
https://doi.org/10.1109/BIBM.2016.7822593

> Enhancers are crucial to the understanding of mechanisms underlying gene transcriptional regulation. Although having been successfully applied in such projects as ENCODE and Roadmap to generate landscape of enhancers in human cell lines, high-throughput biological experimental techniques are still costly and time consuming for even larger scale identification of enhancers across a variety of tissues under different disease status, making computational identification of enhancers indispensable. In this paper, we propose a computational framework, named DeepEnhancer, to classify enhancers from background genomic sequences. We construct convolutional neural networks of various architectures and compare the classification performance with traditional sequence-based classifiers. We first train the deep learning model on the FANTOM5 permissive enhancer dataset, and then fine-tune the model on ENCODE cell type-specific enhancer datasets by adopting the transfer learning strategy. Experimental results demonstrate that DeepEnhancer has superior efficiency and effectiveness in classification tasks, and the use of max-pooling and batch normalization is beneficial to higher accuracy. To make our approach more understandable, we propose a strategy to visualize the convolutional kernels as sequence logos and compare them against the JASPAR database using TOMTOM. In summary, DeepEnhancer allows researchers to train highly accurate deep models and will be broadly applicable in computational biology.

`Study`,  did read in detail, but looks pretty straightforward, CNNs on FANTOM5 and ENCODE data. Also useful for transfer learning.

**Comment by agitter** [comment]
If it is a straightforward CNN we don't need to include it.  We should have plenty of other examples of transfer learning in CNNs.

**Comment by alxndrkalinin** [comment]
@agitter I added it as an example of non-image TL. Also might be useful for @agapow writing `Study:enhancers`.

**Comment by agapow** [comment]
Got it, Thanks.

**Comment by parsboy66** [comment]
hi guys 
how we access to their dataset? i couldnt find any email on their paper to contact them... if anyone knows plz help me tnx in advance

**Comment by cgreene** [comment]
@parsboy66 : you might try the contact info on their department page: http://bioinfo.au.tsinghua.edu.cn/people/

For what it's worth, sharing code, models, and datasets appears to be highly variable at the moment, so there may not be norms that support sharing.

**Comment by parsboy66** [comment]
tnx casey .. i did it and they sent the website addresses of raw datasets
for me
